File: C_Programming/Competitive_Programming/Assignment12/program5.c
//Write a program which accept number from user and count frequency of such a digits  which are less than 6

// Input : 2395
// Output : 3

// Input : 101844
// Output : 5                    

#include<stdio.h>

int CountTwo(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

    while(iNo > 0)
    {
        iDigit = iNo % 10;
        if(iDigit < 6)
        {
            iCnt++;
        }
        iNo = iNo / 10;
    }

    return iCnt;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    iRet = CountTwo(iValue);

    printf("Total Count of Digits which are less than 6 : %d",iRet);
    
    return 0;
}

File: Java_Programming/Competitive_Programming/Assignment21/program5.java
// Write a program to print all number from 1 to N that are divisble by both 2 and 3

class logic 
{
    void printDivisibleBy2and3(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++ ) 
        { 
           if((iCnt % 2 == 0) && (iCnt % 3 == 0))
           {
               System.out.println(iCnt);
           }
        }    
    }
}

class program5
{
    public static void main(String A[] ) 
    {
        logic obj = new logic();
        obj.printDivisibleBy2and3(20);
    }
}
